pkg/controller/efs/filesystem/setup.go

```diff
@@ -93,6 +113,12 @@ func postObserve(_ context.Context, cr *svcapitypes.FileSystem, obj *svcsdk.Desc
 	obs.ConnectionDetails = managed.ConnectionDetails{
 		svcapitypes.ResourceCredentialsSecretIDKey: []byte(meta.GetExternalName(cr)),
 	}
+
+	// BackupPolicy is managed separately from the FileSystem, so we update it here to avoid empty requests in preUpdate
+	updateBackupPolicyErr := e.updateBackupPolicy(ctx, cr, obj)
+	if updateBackupPolicyErr != nil {
+		return managed.ExternalObservation{}, updateBackupPolicyErr
+	}
```

Comment:
I don't have much EFS insights but I think this section should only be executed if `obs.ResourceExists` is true?